src/communication/xcWebSocketSubscriber.js

```diff
@@ -18,19 +18,26 @@ define(["rx"], function (Rx) {
     }
 
 
-	Subscriber.prototype.subscribe = function (componentName, stateMachineName, subscriberListener) {
+	Subscriber.prototype.getFilteredObservable = function (componentName, stateMachineName) {
```

Comment:
We can try to find a better name because it's a public method used by end user.
getStateMachineUpdates ?